add-on/panels/vendor/l10n.js

```diff
@@ -221,14 +221,19 @@
     get: translateString,
 
     // get the document language
-    getLanguage: function() { return gLanguage; },
-
-    // get the direction (ltr|rtl) of the current language
-    getDirection: function() {
-      // http://www.w3.org/International/questions/qa-scripts
-      // Arabic, Hebrew, Farsi, Pashto, Urdu
-      var rtlList = ['ar', 'he', 'fa', 'ps', 'ur'];
-      return (rtlList.indexOf(gLanguage) >= 0) ? 'rtl' : 'ltr';
+    language: {
+      set code(lang) {
+        throw new Error("unsupported");
+      },
+      get code() {
+        return gLanguage;
+      },
+      get direction() {
+        // http://www.w3.org/International/questions/qa-scripts
+        // Arabic, Hebrew, Farsi, Pashto, Urdu
+        var rtlList = ['ar', 'he', 'fa', 'ps', 'ur'];
+        return (rtlList.indexOf(gLanguage) >= 0) ? 'rtl' : 'ltr';
+      }
```

Comment:
I really dig these l10n changes. r=me for those!

One thing however, is that at the top of this file we note that this source came form pdf.js originally. Should we make an effort to keep these in sync, or can we obsolete that statement?